docs/architecture/raft.rst

```diff
@@ -219,6 +219,28 @@ of nodes in the cluster is available. The following examples illustrate how Raft
 
 In summary, Raft makes schema changes safe, but it requires that a quorum of nodes in the cluster is available.
 
+.. _raft-topology-changes:
+
+.. only:: opensource
+
+    Consistent Topology Updates with Raft :label-caution:`Experimental`
+    -------------------------------------------------------------------------------
+
+    ScyllaDB can use Raft for topology updates. With Raft-managed topology enabled, 
+    the cluster can verify that all reachable nodes are using the current topology 
+    and block requests that use the old topology. As a result, topology updates are 
```

Comment:
> the cluster can verify that all reachable nodes are using the current topology and block requests that use the old topology

I would say that this is just a minor feature of raft-based topology (we call it "fencing") and anyway, we still don't allow topology changes without all nodes participating, so everyone needs to get the current topology at each step.

Perhaps the most important feature is: with Raft-managed topology enabled, all topology operations are internally sequenced in a consistent way, with a centralized "topology coordinator" process making sure that topology metadata is synchronized across the nodes on each step of a topology change procedure. This allows safely disabling "ring delay" sleeps that are present in the old gossip-based topology, making topology updates both fast and safe. The cluster administrator can trigger many topology changes concurrently (e.g. start bootstrapping multiple nodes) and the topology coordinator will safely drive all of these changes to completion.